src/SFA.DAS.FAA.Web/Services/VacancyDetailsHelperService.cs

```diff
@@ -67,5 +67,21 @@ public static string GetMapsPostedDate(this DateTime postedDate)
         {
             return $"Posted on {postedDate.ToString("d MMMM", CultureInfo.InvariantCulture)}";
         }
+
+        public static string GetWageText(string wageAmountText)
+        {
+            if (string.IsNullOrEmpty(wageAmountText)) return wageAmountText;
+
+            var poundRemovedStr = wageAmountText.Replace("�", string.Empty);
+
+            if (!decimal.TryParse(poundRemovedStr, out var wageAmount)) return wageAmountText;
+
+            return wageAmount switch
+            {
+                < 100.00M => wageAmount % 1 == 0 ? $"�{wageAmount:#,##} an hour" : $"�{wageAmount:#,##.00} an hour",
+                > 5000.00M => wageAmount % 1 == 0 ? $"�{wageAmount:#,##} a year" : $"�{wageAmount:#,##.00} a year",
```

Comment:
Just to confirm that it's not >= 5000 here?

Reply:
wording from AC

`AC2 Wage context per hour
GIVEN the FAA user is looking at the wage for the NHS jobs advert
WHEN they see a wage that is less than £100 
THEN the value shown will be followed by the words [£xx.xx “an hour”] (confirm words in copy doc)


AC3 Wage context a year
GIVEN the FAA user is looking at the wage for the NHS jobs advert
WHEN they see a wage that is more than £5000 
THEN the value shown will be followed by the words [£xx.xx “a year”] (confirm words in copy doc)`